src/plugins/restAPI/pom.xml

```diff
@@ -8,7 +8,7 @@
     </parent>
     <groupId>org.igniterealtime.openfire.plugins</groupId>
     <artifactId>restAPI</artifactId>
-    <version>1.3.4-SNAPSHOT</version>
+    <version>1.3.5-SNAPSHOT</version>
```

Comment:
@GregDThomas should the version here then be 1.3.5 so that we can release this plugin or ?

Reply:
Yes, I can make this change — makes sense . Thanks for the patience while I learn how the project is run and processes involved .